src/main/java/gr/grnet/pccapi/dto/DomainDto.java

```diff
@@ -1,12 +1,39 @@
 package gr.grnet.pccapi.dto;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
+@Schema(name="DomainResponse", description="An object represents the stored Scientific Domain.")
 public class DomainDto {
 
+    @Schema(
+            type = SchemaType.INTEGER,
+            implementation = String.class,
+            description = "The unique Domain ID generated by the database.",
+            example = "1"
+    )
     public Integer id;
+    @Schema(
+            type = SchemaType.STRING,
+            implementation = String.class,
+            description = "The Domain ID retrieved and generated by the EOSC-Portal.",
```

Comment:
The Domain ID retrieved and generated by EOSC via the EOSC Providers Portal (described in EOSC Profiles)